src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/SelectVariants.java

```diff
@@ -404,6 +400,18 @@
     @Argument(fullName="set-filtered-gt-to-nocall", optional=true, doc="Set filtered genotypes to no-call")
     private boolean setFilteredGenotypesToNocall = false;
 
+    /**
+     * Info annotation fields to be dropped
```

Comment:
Can you note that these are specified based on their keys and not their class names?  When we add annotations we do it by class name, so there's an unavoidable discrepancy there.